test/unit_tests/modules/FlyoutMenu-spec.js

```diff
@@ -0,0 +1,449 @@
+        'when called directly', function() {
+      flyoutMenu.collapse();
+    } );
+  } );
+
+  describe( '.setExpandTransition()', function() {
+    xit( 'should set a transition', function( done ) {
+      flyoutMenu.init();
+      var transition = new MoveTransition( contentDom );
+      flyoutMenu.setExpandTransition( transition, transition.moveLeft );
+      flyoutMenu.addEventListener( 'expandEnd', function() {
+        try {
+          var hasClass = contentDom.classList.contains( 'u-move-transition' );
+          expect( hasClass ).to.be.true;
+          done();
+        } catch( err ) {
+          done( err );
+        }
+      } );
+      triggerDom.click();
+    } );
+  } );
+
+  describe( '.setCollapseTransition()', function() {
+    it( 'should set a transition', function( done ) {
+      flyoutMenu.init();
+      var transition = new MoveTransition( contentDom );
+      triggerDom.click();
+      flyoutMenu.setCollapseTransition( transition, transition.moveLeft );
+      flyoutMenu.addEventListener( 'collapseEnd', function() {
+        try {
+          var hasClass = contentDom.classList.contains( 'u-move-transition' );
+          expect( hasClass ).to.be.true;
+          done();
+        } catch( err ) {
+          done( err );
+        }
+      } );
+      triggerDom.click();
+    } );
+  } );
+
+  describe( '.getTransition()', function() {
+    it( 'should return a transition instance', function() {
+      flyoutMenu.init();
+      expect( flyoutMenu.getTransition() ).to.be.undefined;
+      var transition = new MoveTransition( contentDom );
+      flyoutMenu.setExpandTransition( transition, transition.moveLeft );
+      flyoutMenu.setCollapseTransition( transition, transition.moveToOrigin );
+      expect( flyoutMenu.getTransition() ).to.equal( transition );
+      expect( flyoutMenu.getTransition( FlyoutMenu.COLLAPSE_TYPE ) )
+        .to.equal( transition );
+    } );
+  } );
+
+  describe( '.getDom()', function() {
+    it( 'should return references to full dom', function() {
+      flyoutMenu.init();
+      var dom = flyoutMenu.getDom();
+      expect( dom.container ).to.equal( containerDom );
+      expect( dom.trigger ).to.equal( triggerDom );
+      expect( dom.content ).to.equal( contentDom );
+      expect( dom.altTrigger ).to.equal( altTriggerDom );
+    } );
+  } );
+
+  describe( 'suspend/resume behavior', function() {
+    beforeEach( function() {
+      // Set up expected event listeners.
+      expandBeginSpy = sinon.spy();
+      expandEndSpy = sinon.spy();
+      collapseBeginSpy = sinon.spy();
+      collapseEndSpy = sinon.spy();
+      flyoutMenu.init();
+      flyoutMenu.addEventListener( 'expandBegin', expandBeginSpy );
+      flyoutMenu.addEventListener( 'expandEnd', expandEndSpy );
+      flyoutMenu.addEventListener( 'collapseBegin', collapseBeginSpy );
+      flyoutMenu.addEventListener( 'collapseEnd', collapseEndSpy );
+    } );
+
+    describe( '.resume()', function() {
```

Comment:
Hmm, I wonder if there's another way to do this. Tests shouldn't depend on each other. Rails actually randomizes the running of tests to uncover cross-test dependencies, but I'm not sure how to get around this here without directly re-creating the steps that "suspend" applies. I've re-ordered them in https://github.com/cfpb/cfgov-refresh/pull/1724/commits/722c2ffd2cb08304b2d3f7babdcc3565c8cbf2c1